docs/getting-started/edfi-exchange/guides/how-to-create-a-teamcity-build-for-the-ed-fi-odsapi-suite-3.md

```diff
@@ -0,0 +1,24 @@
+# How To: Create a TeamCity Build for the Ed-Fi ODS/API - Suite 3
+
+![MSDF Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/guides/msdflogo.png)
```

Comment:
This logo is way too big. I'm going to edit it.

Reply:
We also have the same file under `img/edfi-exchange/technology`. I will edit the technology one and would like to delete the duplicate.

Reply:
Edited the technology one. I did not delete the duplicate, will leave that for you. There's also `msdflogo (1).png` in the technology folder.